lib/api.go

```diff
@@ -0,0 +1,119 @@
+package sebak
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"boscoin.io/sebak/lib/error"
+	"boscoin.io/sebak/lib/storage"
+	"github.com/GianlucaGuarini/go-observable"
+	"github.com/gorilla/mux"
+	"boscoin.io/sebak/lib/network"
+)
+
+func AddAPIHandlers(ctx context.Context, t *sebaknetwork.HTTP2Network) {
+
+	t.AddAPIHandler(ctx, "/account/{address}", GetAccountHandler).Methods("GET")
```

Comment:
I think that we may need a bunch of features provided by mux package.
hence, we need `*mux.Route` reference here.

@spikeekips